### PR TITLE
remove transitional filter code

### PR DESCRIPTION
**Pull request by MaxBittker**
sorry for the prettier noise, I commented the three places with real changes (all deletions)
followup as outlined here:
https://github.com/getsentry/sentry/pull/6115/

### CODE REVIEW

File: src/sentry/static/sentry/app/views/projectFilters.jsx [hunk 1]
```diff
@@ -430,33 +425,18 @@ const ProjectFilters = React.createClass({
       localhost: 'Localhost',
       'web-crawlers': 'Web Crawler',
       'invalid-csp': 'Invalid CSP',
-      cors: 'CORS',
-      blacklisted: 'Filtered Events' //TODO(maxbittker) this is only needed until October 10th, 2017
+      cors: 'CORS'
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/projectFilters.jsx]: here

File: src/sentry/static/sentry/app/views/projectFilters.jsx [hunk 2]
```diff
@@ -430,33 +425,18 @@ const ProjectFilters = React.createClass({
...
     };
   },
 
   formatData(rawData) {
-    let cutOverDate = moment([2017, 8, 11]); // date when detailed stats started being recorded
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/projectFilters.jsx]: here

File: src/sentry/static/sentry/app/views/projectFilters.jsx [hunk 3]
```diff
@@ -430,33 +425,18 @@ const ProjectFilters = React.createClass({
...
-
     return Object.keys(this.getStatOpts()).map(stat => {
       return {
         data: rawData[stat].map(([x, y]) => {
           if (y > 0) {
             this.setState({blankStats: false});
           }
 
-          //TODO(maxbittker) this is only needed until October 10th, 2017 :
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/projectFilters.jsx]: and here

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Review by billyvg — APPROVED** [review]
Looks like percy server error caused test failure.

**Pull request merged**